src/printapp/test/test-api.py

```diff
@@ -141,6 +138,27 @@ def test_print(self):
                 response = app.post('/api/print', data=form_data)
                 self.assertEqual(self._status(response), 201)
 
+            response = app.post('/api/deletejob')
+            self.assertEqual(self._status(response), 404)
+            response = app.post('/api/deletejob/')
+            self.assertEqual(self._status(response), 404)
+            response = app.post('/api/deletejob/invalidJobID')
+            self.assertEqual(self._status(response), 200)
+            
+            response = app.get("/api/uniflowstatus")
+            response_json = flask.json.loads(response.data)
+            self.assertEqual(self._status(response), 200)
+            queue = response_json['queue']
+            self.assertIsNotNone(queue)
+            self.assertNotEqual(queue, [])
+            for document in queue:
+                response = app.post('/api/deletejob/' + document['job_id'])
+                self.assertEqual(self._status(response), 200)
+
+            response = app.get("/api/uniflowstatus")
+            response_json = flask.json.loads(response.data)
+            self.assertEqual(response_json['queue'], [])
+
```

Comment:
I just realized there's a problem with the way this is written. If a print submission fails, then the documents won't get deleted from the queue. Is there a way to specify the order tests run in? Because if there is, we could split this into two tests and always (assuming no errors in deleting) have documents get deleted.